packages/marketplace/src/sagas/webhook-subscriptions.ts

```diff
@@ -50,9 +40,9 @@ export const webhookSubscriptionsFetch = function*({ data: applicationId }: Acti
   }
 }
 
-export const webhookTopicsFetch = function*() {
+export const webhookTopicsFetch = function*({ data: applicationId }: Action<string>) {
   try {
-    const response = yield call(fetchWebhookTopic)
+    const response = yield call(fetchWebhookTopic, { applicationId })
     if (response) {
```

Comment:
Here pls remove check if for response because response always return in this block, if error it'll jump to catch block.